src/Cake.Common/Tools/MSBuild/MSBuildSettings.cs

```diff
@@ -38,6 +38,12 @@ public ISet<string> Targets
         public PlatformTarget? PlatformTarget { get; set; }
 
         /// <summary>
+        /// Gets or sets the MSBuild target.
+        /// </summary>
+        /// <value>The MSBuild target.</value>
+        public MSBuildTarget? MSBuildTarget { get; set; }
```

Comment:
Should not be nullable and should be explicitly set to `Automatic` in the constructor.